app/src/main/java/org/fossasia/susi/ai/data/contract/IUtilModel.kt

```diff
@@ -20,8 +20,8 @@ interface IUtilModel {
     fun setServer(isSusiServer: Boolean)
     fun setCustomURL(url: String)
     fun getString(id: Int): String
-    fun getBooleanPref(prefName: String, defaultValue: Boolean): Boolean
-    fun putBooleanPref(prefName: String, value: Boolean)
+    fun getBooleanPref(prefName: Int, defaultValue: Boolean): Boolean
+    fun putBooleanPref(prefName: Int, value: Boolean)
```

Comment:
Good job, instead of replacing these methods, just add two methods here

Reply:
@iamareebjamal umm, but I have changed their implementation and usage everywhere! and their are still some files using from CONSTANTS, would create an issue and send a PR!

Reply:
You just have to add two methods here, no need to change any other file